frontend/src/components/comments/CommentCard.jsx

```diff
@@ -8,101 +8,121 @@ import CommentUpdateForm from "./CommentUpdateForm";
 import CommentLikeApi from "../../api/CommentLikeApi";
 
 export default function CommentCard({
-  postId,
-  comment,
-  onDeleteClick,
-  currentUser,
+    postId,
```

Comment:
Pls could we run the file through VSC formatter?

Reply:
i think it has been updated now please check and let me know!